lib/models/graph-object.js

```diff
@@ -43,6 +43,38 @@ function GraphModelFactory (Model, Constants, configuration) {
             active: function() {
                 var obj = this.toObject();
                 return Constants.Task.ActiveStates.indexOf(obj._status) > -1;
+            },
+            toJSON: function () {
```

Comment:
I don't see the `toJSON` is called in API, so you need submit another PR to make sure the `toJSON` is called.
- https://github.com/RackHD/on-http/blob/a2da546c01dd7176a01f6bc93e8162e5bd1797e8/lib/api/1.1/northbound/workflows.js#L147
- https://github.com/RackHD/on-http/blob/a2da546c01dd7176a01f6bc93e8162e5bd1797e8/lib/api/2.0/workflows.js#L59